src/Components/Input.tsx

```diff
@@ -1,29 +1,184 @@
 import * as React from "react";
-import { View, Text, TextInput, StyleSheet } from "react-native";
+import { StyleSheet, Text, TextInput, Animated } from "react-native";
+import Colors from "../Themes/Colors";
+import { isWeb } from "../Helpers/platform";
 
-// DEMO VERSION
-type InputProps = {
-  field: string;
-  value: string;
-  onChangeText: (value: string) => void;
+type Props = {
+  label: string;
+  onChange(res: { value: string | undefined; isValid: boolean }): void;
+  width: number | string;
+  height: number;
+  onSuccessInputFieldColor: string;
+  secureTextEntry: boolean;
+  validateValue?(value?: string): string | undefined;
+  initialValue?: string;
+  onInputFocus?(): void;
+  onInputBlur?(): void;
+  keyboardType:
```

Comment:
We can reuse the official one
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/981cc6d51eb213943a3606c14ad75d65ea6b360f/types/react-native/index.d.ts#L1207

And we can use `?:` instead of undefined
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/981cc6d51eb213943a3606c14ad75d65ea6b360f/types/react-native/index.d.ts#L1348

Reply:
https://github.com/trucknet-io/react-native-web-uikit/pull/12#discussion_r313068890

Reply:
And we can use ?: instead of undefined - I know about this, but we don't need this here when we define prop in defaultProps

Reply:
however it's just copy of react native `keyboardType`